inst/helpfiles/help.md

```diff
@@ -10,7 +10,7 @@
 
 Each tab has its own help information which can be accessed by clicking on the question mark circle at the top of the instruction box of the tab. 
 
-For a more detailed user guide go to XXXX (insert link here).
+For a more detailed user guide go to [this link](https://poissonconsulting.github.io/bisonpicsuite/bisonpic-user-guide.html).
```

Comment:
When doing hyper links the recommendation to make them more accessibility is to not use  generic phrases like "click here" or "read more," and instead use specific descriptions. For example, instead of saying "click here to view our product catalog," write "view our product catalog" as the hyperlink.

This comes from how screen readers for visually impaired people read the html code out to them.